ml-stat/ml-stat-tests.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference path="./index.d.ts" />

```

Comment:
Why compiler complains every time that cannot find the module if I remove that line?